design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
+- Please post the GitHub PR link to `#design-review:matrix.mathesar.org`.
+- If the reviewers request changes or have questions, please make the changes and re-request the review.
+- Once everyone has had a chance to review (or the timeline expires) and you're satisfied with the feedback, update the spec and ensure that it all fits together nicely.
+
+### Reviewers
+
+- Check for outstanding [design spec review requests](https://github.com/centerofci/mathesar-wiki/pulls?q=is%3Aopen) at least once a day.
+- Comment on the GitHub PR with feedback. Please follow the general advice above.
+Once you're done reviewing the document, approve the changes or unassign yourself from the PR if you don't have any feedback.
+- Request changes explicitly if you want them included in the spec before it is finalized.
 - By sharing feedback, you actively participate in the UX process; make sure you read the context documents and are clear on the user's needs. If needed, ask questions, don't make assumptions.
 
-# Advice for Reviewers
+#### Advice for Reviewers
+
 - Describe the problem with the design but don't assume a specific design solution
 - Strive to think from the user's perspective, not your own
 - If you find inconsistencies in the design, describe what you expected to encounter instead
 - Adapt your feedback to the design's fidelity level
-    - e.g. pointing out alignment issues on wireframes is not useful because they are not meant to be high-fidelity
+  - e.g., pointing out alignment issues on wireframes is not useful because they are not meant to be high-fidelity
 - Avoid ambiguity, try to articulate problems clearly and concisely
-    - Use descriptive words such as "clear", "helpful", "obvious", "confusing", "complex", rather than "bad", "wrong", "off", etc.
-- Think backwards from the end-goal
-- Don't highlight only negative aspects of the design. Positive feedback provides valuable information.
\ No newline at end of file
+  - Use descriptive words such as "clear", "helpful", "obvious", "confusing", "complex", rather than "bad", "wrong", "off", etc.
+- Think backward from the end-goal
+- Don't highlight only the negative aspects of the design. Positive feedback provides valuable information.
+
+## Spec Content
+
+The spec content will depend on the particular design problem being addressed. However, it is recommended that you include the following parts:
+
+- An overview or summary of the problem.
+  - Assume the person looking at the spec didn't read the related issue.
+- User Scenarios.
+  - Go from simplest to complex so that new details are added incrementally, for example, 'User opens a table' to 'User opens multiple tables at once.
+- A link to a prototype or wireframes or any artifact used to demonstrate the design solution.
+  - Including video walkthroughs for prototypes or wireframes can help the team better understand the proposed solution.
```

Comment:
maybe mention that we have a Loom account you can request access to if you don't have access already.